2021-10-03 11:27:33,462 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 11:27:33,465 - job_service_app - INFO - clientname: Dina
2021-10-03 11:27:33,465 - job_service_app - INFO - Going to call add job API...
2021-10-03 11:27:33,465 - job_service - INFO - In Add Job API
2021-10-03 11:27:33,465 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 11:27:33,539 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 50, in addjobDAO
    cursor.execute(mySql_Add_Job_insert_Query)
AttributeError: 'NoneType' object has no attribute 'execute'
2021-10-03 11:27:33,544 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 11:27:33] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 11:34:48,286 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 11:34:48,286 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 11:35:02,886 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 11:35:02,886 - job_service_app - INFO - clientname: Dina
2021-10-03 11:35:02,886 - job_service_app - INFO - Going to call add job API...
2021-10-03 11:35:02,886 - job_service - INFO - In Add Job API
2021-10-03 11:35:02,886 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 11:35:02,886 - job_dao - INFO - In Add Job DAO Service
2021-10-03 11:35:02,886 - job_dao - INFO - jobid 373213clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 11:35:02,904 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 50, in addjobDAO
    cursor.execute(mySql_Add_Job_insert_Query)
AttributeError: 'NoneType' object has no attribute 'execute'
2021-10-03 11:35:02,908 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 11:35:02] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 11:39:25,237 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 11:39:25,237 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 11:39:30,558 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 11:39:30,558 - job_service_app - INFO - clientname: Dina
2021-10-03 11:39:30,558 - job_service_app - INFO - Going to call add job API...
2021-10-03 11:39:30,558 - job_service - INFO - In Add Job API
2021-10-03 11:39:30,558 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 11:39:30,558 - job_dao - INFO - In Add Job DAO Service
2021-10-03 11:39:30,558 - job_dao - INFO - jobid 876528 clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 11:39:30,572 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 52, in addjobDAO
    cursor.execute(mySql_Add_Job_insert_Query)
AttributeError: 'NoneType' object has no attribute 'execute'
2021-10-03 11:39:30,574 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 11:39:30] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 11:41:22,535 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 11:41:22,535 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 11:41:27,603 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 11:41:27,603 - job_service_app - INFO - clientname: Dina
2021-10-03 11:41:27,603 - job_service_app - INFO - Going to call add job API...
2021-10-03 11:41:27,603 - job_service - INFO - In Add Job API
2021-10-03 11:41:27,603 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 11:41:27,603 - job_dao - INFO - In Add Job DAO Service
2021-10-03 11:41:27,603 - job_dao - INFO - jobid 336272 clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 11:41:27,625 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 52, in addjobDAO
    cursor.execute(mySql_Add_Job_insert_Query)
AttributeError: 'NoneType' object has no attribute 'execute'
2021-10-03 11:41:27,630 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 11:41:27] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 11:45:36,943 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 11:45:36,943 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 11:45:44,998 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 11:45:44,998 - job_service_app - INFO - clientname: Dina
2021-10-03 11:45:44,998 - job_service_app - INFO - Going to call add job API...
2021-10-03 11:45:44,998 - job_service - INFO - In Add Job API
2021-10-03 11:45:44,998 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 11:45:44,998 - job_dao - INFO - In Add Job DAO Service
2021-10-03 11:45:44,998 - job_dao - INFO - jobid 104419 clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 11:45:45,032 - job_dao - INFO - In Finally of DB Connection
2021-10-03 11:45:45,033 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 54, in addjobDAO
    cursor.execute(mySql_Add_Job_insert_Query)
AttributeError: 'NoneType' object has no attribute 'execute'
2021-10-03 11:45:45,033 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 11:45:45] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 12:12:34,031 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 12:12:34,031 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 12:12:41,606 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 12:12:41,607 - job_service_app - INFO - clientname: Dina
2021-10-03 12:12:41,607 - job_service_app - INFO - Going to call add job API...
2021-10-03 12:12:41,607 - job_service - INFO - In Add Job API
2021-10-03 12:12:41,607 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 12:12:41,607 - job_dao - INFO - In Add Job DAO Service
2021-10-03 12:12:41,607 - job_dao - INFO - jobid 915674 clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 12:12:41,634 - job_dao - INFO - In Finally of DB Connection
2021-10-03 12:12:41,635 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 54, in addjobDAO
    cursor.execute(mySql_Add_Job_insert_Query)
AttributeError: 'NoneType' object has no attribute 'execute'
2021-10-03 12:12:41,635 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 12:12:41] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 12:13:21,839 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 12:13:21,839 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 12:13:24,602 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 12:13:24,602 - job_service_app - INFO - clientname: Dina
2021-10-03 12:13:24,602 - job_service_app - INFO - Going to call add job API...
2021-10-03 12:13:24,602 - job_service - INFO - In Add Job API
2021-10-03 12:13:24,603 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 12:13:24,603 - job_dao - INFO - In Add Job DAO Service
2021-10-03 12:13:24,603 - job_dao - INFO - jobid 891542 clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 12:13:24,670 - job_dao - INFO - In Finally of DB Connection
2021-10-03 12:13:24,670 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 54, in addjobDAO
    cursor.execute(mySql_Add_Job_insert_Query)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/mysql/connector/cursor.py", line 535, in execute
    if not self._connection:
ReferenceError: weakly-referenced object no longer exists
2021-10-03 12:13:24,673 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 12:13:24] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 12:34:44,726 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 12:34:44,726 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 12:35:00,164 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 12:35:00,164 - job_service_app - INFO - clientname: Dina
2021-10-03 12:35:00,164 - job_service_app - INFO - Going to call add job API...
2021-10-03 12:35:00,164 - job_service - INFO - In Add Job API
2021-10-03 12:35:00,164 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 12:35:00,164 - job_dao - INFO - In Add Job DAO Service
2021-10-03 12:35:00,164 - job_dao - INFO - jobid 779395 clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 12:35:00,193 - job_dao - INFO - In Finally of DB Connection
2021-10-03 12:35:00,194 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 62, in addjobDAO
    connection.execute(mySql_Add_Job_insert_Query)
AttributeError: 'mySQLConnection' object has no attribute 'execute'
2021-10-03 12:35:00,194 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 12:35:00] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 12:35:43,736 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 12:35:43,736 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 12:35:47,541 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 12:35:47,542 - job_service_app - INFO - clientname: Dina
2021-10-03 12:35:47,542 - job_service_app - INFO - Going to call add job API...
2021-10-03 12:35:47,542 - job_service - INFO - In Add Job API
2021-10-03 12:35:47,542 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 12:35:47,542 - job_dao - INFO - In Add Job DAO Service
2021-10-03 12:35:47,542 - job_dao - INFO - jobid 801342 clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 12:35:47,568 - job_dao - INFO - In Finally of DB Connection
2021-10-03 12:35:47,568 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 62, in addjobDAO
    connection.executeQuery(mySql_Add_Job_insert_Query)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 48, in executeQuery
    self.connection.cursor.execute(query)
AttributeError: 'function' object has no attribute 'execute'
2021-10-03 12:35:47,569 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 12:35:47] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 12:45:58,602 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 12:45:58,603 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 12:46:26,924 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'adf'}
2021-10-03 12:46:26,925 - job_service_app - INFO - clientname: Dina
2021-10-03 12:46:26,925 - job_service_app - INFO - Going to call add job API...
2021-10-03 12:46:26,925 - job_service - INFO - In Add Job API
2021-10-03 12:46:26,925 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements adf
2021-10-03 12:46:26,925 - job_dao - INFO - In Add Job DAO Service
2021-10-03 12:46:26,925 - job_dao - INFO - jobid 719783 clientname Dina jobprofile Staff2 jobrequirements adf
2021-10-03 12:46:26,925 - job_dao - INFO - insert into jobs values(719783,Dina,Staff2,adf)
2021-10-03 12:46:26,925 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 61, in addjobDAO
    connection.executeQuery(mySql_Add_Job_insert_Query)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 46, in executeQuery
    self.connection.cursor.execute(query)
AttributeError: 'function' object has no attribute 'execute'
2021-10-03 12:46:26,929 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 12:46:26] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 13:04:05,342 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:04:05,343 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:04:18,237 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'adf'}
2021-10-03 13:04:18,237 - job_service_app - INFO - clientname: Dina
2021-10-03 13:04:18,237 - job_service_app - INFO - Going to call add job API...
2021-10-03 13:04:18,237 - job_service - INFO - In Add Job API
2021-10-03 13:04:18,237 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements adf
2021-10-03 13:04:18,238 - job_dao - INFO - In Add Job DAO Service
2021-10-03 13:04:18,238 - job_dao - INFO - jobid 864480 clientname Dina jobprofile Staff2 jobrequirements adf
2021-10-03 13:04:18,275 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:04:18,279 - job_dao - INFO - Query executed successfully
2021-10-03 13:04:18,279 - job_dao - INFO - Successfully inserted the job entry for the id 864480
2021-10-03 13:04:18,280 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:04:18] "POST /jobservice/addjob HTTP/1.1" 200 -
2021-10-03 13:08:27,360 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:08:27,360 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:08:46,455 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'adf'}
2021-10-03 13:08:46,455 - job_service_app - INFO - clientname: Dina
2021-10-03 13:08:46,455 - job_service_app - INFO - Going to call add job API...
2021-10-03 13:08:46,455 - job_service - INFO - In Add Job API
2021-10-03 13:08:46,455 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements adf
2021-10-03 13:08:46,455 - job_dao - INFO - In Add Job DAO Service
2021-10-03 13:08:46,455 - job_dao - INFO - jobid 898772 clientname Dina jobprofile Staff2 jobrequirements adf
2021-10-03 13:08:46,491 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:08:46,491 - job_service_app - ERROR - Exception on /jobservice/addjob [POST]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 31, in add_candidate
    jobAdded =  job_service.addJob(clientname=clientname, jobprofile=jobprofile, jobrequirements=jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 23, in addJob
    job_dao.addjobDAO(clientname, jobprofile, jobrequirements)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 75, in addjobDAO
    executeDBQuery(insert_stmt, data)
TypeError: executeDBQuery() missing 1 required positional argument: 'data'
2021-10-03 13:08:46,491 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:08:46] "[35m[1mPOST /jobservice/addjob HTTP/1.1[0m" 500 -
2021-10-03 13:09:33,879 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:09:33,880 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:09:39,832 - job_service_app - INFO - JSON Request: {'clientname': 'Dina', 'jobprofile': 'Staff2', 'jobrequirements': 'Java Cloud'}
2021-10-03 13:09:39,832 - job_service_app - INFO - clientname: Dina
2021-10-03 13:09:39,833 - job_service_app - INFO - Going to call add job API...
2021-10-03 13:09:39,833 - job_service - INFO - In Add Job API
2021-10-03 13:09:39,833 - job_service - INFO - clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 13:09:39,833 - job_dao - INFO - In Add Job DAO Service
2021-10-03 13:09:39,833 - job_dao - INFO - jobid 917031 clientname Dina jobprofile Staff2 jobrequirements Java Cloud
2021-10-03 13:09:39,865 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:09:39,873 - job_dao - INFO - Query executed successfully
2021-10-03 13:09:39,873 - job_dao - INFO - Successfully inserted the job entry for the id 917031
2021-10-03 13:09:39,874 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:09:39] "POST /jobservice/addjob HTTP/1.1" 200 -
2021-10-03 13:12:46,880 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:12:46,880 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:12:52,407 - job_service - INFO - In Get All Jobs API
2021-10-03 13:12:52,407 - job_dao - INFO - In Add Job DB Service
2021-10-03 13:12:52,438 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:12:52,438 - job_service_app - ERROR - Exception on /jobservice/getalljobs [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 37, in get_All_Candidates
    jobsList = job_service.getAllJobs()
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 33, in getAllJobs
    jobList = job_dao.getJobsDAO()
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 83, in getJobsDAO
    result = executeDBQuery(connection, mySql_Get_Jobs_Select_Query)
TypeError: executeDBQuery() missing 1 required positional argument: 'data'
2021-10-03 13:12:52,439 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:12:52] "[35m[1mGET /jobservice/getalljobs HTTP/1.1[0m" 500 -
2021-10-03 13:14:37,099 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:14:37,099 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:14:42,193 - job_service - INFO - In Get All Jobs API
2021-10-03 13:14:42,193 - job_dao - INFO - In Add Job DB Service
2021-10-03 13:14:42,226 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:14:42,230 - job_dao - INFO - Query execution error 'Unread result found' occurred
2021-10-03 13:14:42,230 - job_dao - INFO - Successfully get the jobs None
2021-10-03 13:14:42,230 - job_service - INFO - Get jobs from DB is null
2021-10-03 13:14:42,231 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:14:42] "GET /jobservice/getalljobs HTTP/1.1" 200 -
2021-10-03 13:19:30,011 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:19:30,011 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:19:39,802 - job_service - INFO - In Get All Jobs API
2021-10-03 13:19:39,802 - job_dao - INFO - In Add Job DB Service
2021-10-03 13:19:39,872 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:19:39,875 - job_service_app - ERROR - Exception on /jobservice/getalljobs [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 37, in get_All_Candidates
    jobsList = job_service.getAllJobs()
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 33, in getAllJobs
    jobList = job_dao.getJobsDAO()
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 92, in getJobsDAO
    result = executeDBExecuteQuery(connection, mySql_Get_Jobs_Select_Query)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_dao.py", line 58, in executeDBExecuteQuery
    result = cursor.execute(statement, data)
NameError: name 'data' is not defined
2021-10-03 13:19:39,875 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:19:39] "[35m[1mGET /jobservice/getalljobs HTTP/1.1[0m" 500 -
2021-10-03 13:20:39,469 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:20:39,469 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:20:42,890 - job_service - INFO - In Get All Jobs API
2021-10-03 13:20:42,890 - job_dao - INFO - In Add Job DB Service
2021-10-03 13:20:42,920 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:20:42,923 - job_dao - INFO - Query executed successfully select * from jobs
2021-10-03 13:20:42,923 - job_dao - INFO - Successfully get the jobs None
2021-10-03 13:20:42,923 - job_service - INFO - Get jobs from DB is null
2021-10-03 13:20:42,924 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:20:42] "GET /jobservice/getalljobs HTTP/1.1" 200 -
2021-10-03 13:21:57,492 - job_service_app - INFO - JSON Request: {'clientname': 'Ram', 'jobprofile': 'Senior Staff', 'jobrequirements': 'Cloud'}
2021-10-03 13:21:57,492 - job_service_app - INFO - clientname: Ram
2021-10-03 13:21:57,492 - job_service_app - INFO - Going to call add job API...
2021-10-03 13:21:57,493 - job_service - INFO - In Add Job API
2021-10-03 13:21:57,493 - job_service - INFO - clientname Ram jobprofile Senior Staff jobrequirements Cloud
2021-10-03 13:21:57,493 - job_dao - INFO - In Add Job DAO Service
2021-10-03 13:21:57,493 - job_dao - INFO - jobid 265905 clientname Ram jobprofile Senior Staff jobrequirements Cloud
2021-10-03 13:21:57,519 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:21:57,529 - job_dao - INFO - Query executed successfully INSERT INTO jobs (jobid, clientname, jobprofile, jobrequirements) VALUES (%s, %s, %s, %s)
2021-10-03 13:21:57,529 - job_dao - INFO - Successfully inserted the job entry for the id 265905
2021-10-03 13:21:57,530 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:21:57] "POST /jobservice/addjob HTTP/1.1" 200 -
2021-10-03 13:22:47,629 - job_service - INFO - In Get All Jobs API
2021-10-03 13:22:47,629 - job_dao - INFO - In Add Job DB Service
2021-10-03 13:22:47,653 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:22:47,656 - job_dao - INFO - Query executed successfully select * from jobs
2021-10-03 13:22:47,656 - job_dao - INFO - Successfully get the jobs None
2021-10-03 13:22:47,657 - job_service - INFO - Get jobs from DB is null
2021-10-03 13:22:47,657 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:22:47] "GET /jobservice/getalljobs HTTP/1.1" 200 -
2021-10-03 13:25:14,444 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:25:14,444 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:25:20,783 - job_service - INFO - In Get All Jobs API
2021-10-03 13:25:20,783 - job_dao - INFO - In Add Job DB Service
2021-10-03 13:25:20,814 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:25:20,817 - job_dao - INFO - Query executed successfully select * from jobs
2021-10-03 13:25:20,818 - job_dao - INFO - Successfully get the jobs [(917031, 'Dina', 'Staff2', 'Java Cloud'), (265905, 'Ram', 'Senior Staff', 'Cloud')]
2021-10-03 13:25:20,818 - job_service - INFO - Get jobs from DB is [[917031, "Dina", "Staff2", "Java Cloud"], [265905, "Ram", "Senior Staff", "Cloud"]]
2021-10-03 13:25:20,818 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:25:20] "GET /jobservice/getalljobs HTTP/1.1" 200 -
2021-10-03 13:41:12,258 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:41:12,258 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:41:19,981 - job_service - INFO - In Get All Jobs API
2021-10-03 13:41:19,981 - job_dao - INFO - In Add Job DB Service
2021-10-03 13:41:20,013 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:41:20,016 - job_dao - INFO - Query executed successfully select * from jobs
2021-10-03 13:41:20,016 - job_dao - INFO - Successfully get the jobs [(917031, 'Dina', 'Staff2', 'Java Cloud'), (265905, 'Ram', 'Senior Staff', 'Cloud')]
2021-10-03 13:41:20,017 - job_service_app - ERROR - Exception on /jobservice/getalljobs [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service_app.py", line 37, in get_All_Candidates
    jobsList = job_service.getAllJobs()
  File "/Users/dgnaneswaran/OneDrive - VMware, Inc/Dina/VM Materials/Take1/Cloud Architect Course works/Cloud Architect Final Project/Recruting-Agency/recruitment-job-service/job_service.py", line 35, in getAllJobs
    job = dict({'jobid': row[1], 'clientname': row[2], 'jobprofile': row[3], 'jobrequirements': row[4]})
IndexError: tuple index out of range
2021-10-03 13:41:20,017 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:41:20] "[35m[1mGET /jobservice/getalljobs HTTP/1.1[0m" 500 -
2021-10-03 13:42:18,535 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 13:42:18,536 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
2021-10-03 13:42:22,067 - job_service - INFO - In Get All Jobs API
2021-10-03 13:42:22,067 - job_dao - INFO - In Add Job DB Service
2021-10-03 13:42:22,099 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:42:22,103 - job_dao - INFO - Query executed successfully select * from jobs
2021-10-03 13:42:22,103 - job_dao - INFO - Successfully get the jobs [(917031, 'Dina', 'Staff2', 'Java Cloud'), (265905, 'Ram', 'Senior Staff', 'Cloud')]
2021-10-03 13:42:22,104 - job_service - INFO - Get jobs from DB is [{"jobid": 917031, "clientname": "Dina", "jobprofile": "Staff2", "jobrequirements": "Java Cloud"}, {"jobid": 265905, "clientname": "Ram", "jobprofile": "Senior Staff", "jobrequirements": "Cloud"}]
2021-10-03 13:42:22,104 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:42:22] "GET /jobservice/getalljobs HTTP/1.1" 200 -
2021-10-03 13:43:53,921 - job_service_app - INFO - JSON Request: {'clientname': 'Ramu', 'jobprofile': 'Sr MTS', 'jobrequirements': 'Python'}
2021-10-03 13:43:53,922 - job_service_app - INFO - clientname: Ramu
2021-10-03 13:43:53,922 - job_service_app - INFO - Going to call add job API...
2021-10-03 13:43:53,922 - job_service - INFO - In Add Job API
2021-10-03 13:43:53,922 - job_service - INFO - clientname Ramu jobprofile Sr MTS jobrequirements Python
2021-10-03 13:43:53,922 - job_dao - INFO - In Add Job DAO Service
2021-10-03 13:43:53,922 - job_dao - INFO - jobid 39748 clientname Ramu jobprofile Sr MTS jobrequirements Python
2021-10-03 13:43:54,006 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:43:54,015 - job_dao - INFO - Query executed successfully INSERT INTO jobs (jobid, clientname, jobprofile, jobrequirements) VALUES (%s, %s, %s, %s)
2021-10-03 13:43:54,015 - job_dao - INFO - Successfully inserted the job entry for the id 39748
2021-10-03 13:43:54,016 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:43:54] "POST /jobservice/addjob HTTP/1.1" 200 -
2021-10-03 13:43:59,430 - job_service - INFO - In Get All Jobs API
2021-10-03 13:43:59,430 - job_dao - INFO - In Add Job DB Service
2021-10-03 13:43:59,454 - job_dao - INFO - In Finally of DB Connection
2021-10-03 13:43:59,458 - job_dao - INFO - Query executed successfully select * from jobs
2021-10-03 13:43:59,458 - job_dao - INFO - Successfully get the jobs [(917031, 'Dina', 'Staff2', 'Java Cloud'), (265905, 'Ram', 'Senior Staff', 'Cloud'), (39748, 'Ramu', 'Sr MTS', 'Python')]
2021-10-03 13:43:59,458 - job_service - INFO - Get jobs from DB is [{"jobid": 917031, "clientname": "Dina", "jobprofile": "Staff2", "jobrequirements": "Java Cloud"}, {"jobid": 265905, "clientname": "Ram", "jobprofile": "Senior Staff", "jobrequirements": "Cloud"}, {"jobid": 39748, "clientname": "Ramu", "jobprofile": "Sr MTS", "jobrequirements": "Python"}]
2021-10-03 13:43:59,458 - werkzeug - INFO - 127.0.0.1 - - [03/Oct/2021 13:43:59] "GET /jobservice/getalljobs HTTP/1.1" 200 -
2021-10-03 14:36:11,136 - werkzeug - WARNING -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-10-03 14:36:11,136 - werkzeug - INFO -  * Running on http://10.104.68.227:9000/ (Press CTRL+C to quit)
